build.gradle

```diff
@@ -1,24 +1,15 @@
-buildscript {
-    repositories {
-        mavenCentral()
-        maven { url "https://repo.grails.org/grails/core" }
-    }
-    dependencies {
-        classpath "org.grails:grails-gradle-plugin:$grailsGradlePluginVersion"
-        classpath 'com.adarshr:gradle-test-logger-plugin:4.0.0'
-    }
-}
-
-plugins {
-    id 'idea'
-    id 'io.github.gradle-nexus.publish-plugin' version '1.3.0'
-}
-
 allprojects {
-    apply plugin:"idea"
+    apply plugin: 'idea'
```

Comment:
Add below task for this project

1. bootrun block
2. sourceJar
3. packageJavadoc
4. GroovyCompile
5. jar
6. Test

Reply:
bootRun task added and for sourceJar, packageJavaDoc,GroovyCompile already task utilized default one and test junit is already available.